tools/downloader/downloader.py

```diff
@@ -191,6 +191,8 @@ def main():
         help='download only models whose names match at least one of the patterns in the specified file')
     parser.add_argument('--all',  action='store_true', help='download all available models')
     parser.add_argument('--print_all', action='store_true', help='print all available models')
+    parser.add_argument('--precisions', metavar='PREC[,PREC...]',
```

Comment:
why not `nargs`?  it allows setting list of arguments using space as delimiter. 
https://docs.python.org/3/library/argparse.html#nargs

Reply:
To align it with `converter.py`:

https://github.com/opencv/open_model_zoo/blob/c3b4b2b9845db2ca15e03df9804b6f98d7a3c211/tools/downloader/converter.py#L98-L99